### PR TITLE
Update customer-key-set-up.md

### PR DESCRIPTION
**Pull request by msbemba**
Thales sold the nCipher business to Entrust Datacard so changing Thales nShield HSMs to nCipher nShield HSMs

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2369

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/customer-key-set-up.md [hunk 1]
```diff
@@ -275,13 +275,13 @@ For example,
 Add-AzKeyVaultKey -VaultName Contoso-O365EX-NA-VaultA1 -Name Contoso-O365EX-NA-VaultA1-Key001 -Destination Software -KeyOps wrapKey,unwrapKey
 ```
 
-To import a key directly into your key vault, you need to have a Thales nShield Hardware Security Module.
+To import a key directly into your key vault, you need to have a nCipher nShield Hardware Security Module.
   
 Some organizations prefer this approach to establish the provenance of their keys, and the this method also provides the following:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md]:
```suggestion
Some organizations prefer this approach to establish the provenance of their keys, and then this method also provides the following:
```

File: microsoft-365/compliance/customer-key-set-up.md [hunk 2]
```diff
@@ -275,13 +275,13 @@ For example,
...
   
-- The tool set used for import includes attestation from Thales that the Key Exchange Key (KEK) that is used to encrypt the key you generate is not exportable and is generated inside a genuine HSM that was manufactured by Thales.
+- The tool set used for import includes attestation from nCipher that the Key Exchange Key (KEK) that is used to encrypt the key you generate is not exportable and is generated inside a genuine HSM that was manufactured by nCipher.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md]:
```suggestion
- The toolset used for import includes attestation from nCipher that the Key Exchange Key (KEK) that is used to encrypt the key you generate is not exportable and is generated inside a genuine HSM that was manufactured by nCipher.
```

File: microsoft-365/compliance/customer-key-set-up.md [hunk 3]
```diff
@@ -275,13 +275,13 @@ For example,
...
 
-- The tool set includes attestation from Thales that the Azure Key Vault security world was also generated on a genuine HSM manufactured by Thales. This attestation proves to you that Microsoft is also using genuine Thales hardware.
+- The tool set includes attestation from nCipher that the Azure Key Vault security world was also generated on a genuine HSM manufactured by nCipher. This attestation proves to you that Microsoft is also using genuine nCipher hardware.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md]:
```suggestion
- The toolset includes attestation from nCipher that the Azure Key Vault security world was also generated on a genuine HSM manufactured by nCipher. This attestation proves to you that Microsoft is also using genuine nCipher hardware.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few additional edits please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Pull request merged**